src/core/Akka.Docs.Tests/Clustet.Metrics/RoutersUsageSample.cs

```diff
@@ -97,6 +100,6 @@ private void SendJobs()
             }
         }
     }
-
+    // </FactorialFrontend>
```

Comment:
Oh wow, this works without using `#region`s ?

Reply:
Yup, it should, according to this description: https://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html#tag-name-representation-in-code-snippet-source-file

Reply:
As I said below, could not check if all this references work properly...

Reply:
Guess we'll find out - would love to get rid of regions elsewhere.